internal/connection.go

```diff
@@ -211,7 +219,7 @@ func (c *Connection) close(err error) {
 	if !atomic.CompareAndSwapInt32(&c.status, 0, 1) {
 		return
 	}
-	//TODO log close message
+	c.logger.Debug("Connection :", c, " closed, err: ", err)
```

Comment:
you are logging different messages from what Java logs (both content and number of messages), we need to log at least for info level and above the ones that Java client logs.

Reply:
There are some parts like that but to be able to get the same message as Java it requires some more work. So I think they can be added in another PR.